lib/quicly.c

```diff
@@ -2570,6 +2571,10 @@ static quicly_conn_t *create_connection(quicly_context_t *ctx, uint32_t protocol
     conn->egress.send_ack_at = INT64_MAX;
     conn->egress.send_probe_at = INT64_MAX;
     conn->super.ctx->init_cc->cb(conn->super.ctx->init_cc, &conn->egress.cc, initcwnd, conn->stash.now);
+    if (conn->super.ctx->cc_slowstart != NULL) {
+        conn->egress.cc.type->cc_slowstart->name = conn->super.ctx->cc_slowstart->name;
```

Comment:
These manual updates should be changed to the quicly_set_ss function provided via patch file in my previous email

Reply:
No need at this point, one can do the same thing by providing a different context.